screens/RequestStatus.js

```diff
@@ -0,0 +1,191 @@
+
+let colors = ['#605DF1', '#6DC4E0']
+
+let items = [
+  {
+    name: 'Apple',
+    quantity: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  },
+  {
+    name: 'Apple',
+    quantity: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  },
+  {
+    name: 'Apple',
+    quantity: 3,
+    comment: 'Granny Smith',
+    check: require('./../assets/images/checkbox.png'),
+  }
+]
+
+export default class RequestStatus extends React.Component { 
+  static navigationOptions = {
+    header: null,
+    };
+    
+  
+  render() {
+    return (
+        <View style={styles.container}>
+            <OrangeBackground/>
+            <Toolbar pageType={'Driver'} marginBottom={-Dimensions.get('screen').height*.02}/>
+            
+            <View style={styles.subView}>
+              <Image style={styles.arrowIcon}
+                  source={require('./../assets/images/back_arrow.png')} />
+              <Text style={styles.browseText}>Browse Other Options</Text>
+            </View>
+            <View style={styles.mainView}>
+              <View style={styles.whiteCard}>
+                  <View style={styles.nameBox}>
+                    <Text style={styles.nameText}>Bob - CVS</Text>
+                  </View>
+                  <View style={[styles.tableRows, {marginTop: Dimensions.get('screen').height*0.06, backgroundColor: '#6DC4E0'}]}>
+                    <Text style={styles.itemInfo}>Item</Text>
+                    <Text style={styles.itemInfo}>Q</Text>
+                    <Text style={styles.itemInfo}>Comments</Text>
+                    <Image style={styles.checkIcon}
+                        source={require('./../assets/images/check.png')} />
+                  </View>
+                  <View style={styles.table}>
+                      <FlatList
+                      data={items}
+                      keyExtractor={(item, index) => index.toString()}
+                      renderItem={
+                        ({item, index}) => {
+                        return (
+                            <View style={[styles.tableRows, {backgroundColor: colors[index % colors.length]}]}>
+                                <Text style={styles.itemsText}>{item.name}</Text>
+                                <Text style={styles.itemsText}>{item.quantity}</Text>
+                                <Text style={styles.itemsText}>{item.comment}</Text>
+                                <Image style={styles.checkIcon}
+                                    source={item.check} />
+                            </View>
+                      )
+                      }
+                    }
+                    />
+                  </View>
+                  <View style={{marginTop: -Dimensions.get('screen').height*.04}}>
+                    <Text style={styles.dropText}>Phone: 123-456-789</Text>
+                    <Text style={styles.dropText}>Email: ab.22@dartmouth.edu</Text>
```

Comment:
The 'Email:' should be in its own Text component while the actual email should be in its own component.